iac/main.tf

```diff
@@ -15,7 +15,15 @@ terraform {
 
```

Comment:
Infrastructure Testing Advice:
1.  **Static Analysis**: Consider adding `terraform validate` and `tflint` to your CI pipeline to catch syntax and best-practice issues early.
2.  **Integration Tests**: For more robust testing, look into **Terratest** (a Go library) to spin up real resources, verify them, and tear them down in a test environment.